libs/frontend/feature-render-builder/src/lib/compute-builder-reusable-styles.ts

```diff
@@ -14,10 +14,10 @@ export const computeBuilderReusableStyles = (site: ISite): IRawStyleRecord => {
   const accumulatedBreakpointIds = new Set<string>([DEFAULT_BREAKPOINT_ID])
 
   for (const breakpoint of descSortedBreakpoints.value) {
+    accumulatedBreakpointIds.add(breakpoint.id)
```

Comment:
@abemscac I had to move this line above the `activeBreakpoint`, otherwise the current breakpoint wouldn't be added.